src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -219,6 +218,27 @@ public CreatePaymentResponse resumePayment(
             .build();
     }
 
+    @LogExecutionTime
+    @Transactional(transactionManager = "transactionManager")
+    public Claim saveCitizenClaim(
+        String authorisation,
+        ClaimData claimData,
```

Comment:
We are getting this ClaimData from our endpoint, but we are not storing this in CCD.

Reply:
yeah, I was imagining the claim data to be the same as the one we already have stored in ccd after the initiate/resume payment. But you're right, let's save it too

Reply:
Thanks